test/component-can-bind-test.js

```diff
@@ -0,0 +1,66 @@
+var Bind = require("can-bind");
+var Component = require("can-component");
+var QUnit = require("steal-qunit");
+var SimpleMap = require("can-simple-map");
+var stache = require("can-stache");
+var value = require("can-value");
+
+QUnit.module("can-component integration with can-bind");
+
+QUnit.test("Using can-bind in connectedCallback works as documented", function(assert) {
```

Comment:
This test is similar to the component shown as an example in can-bind’s docs; see here on the 5.0 site: https://canjs.github.io/next/doc/can-bind.html#Use